apps/src/code-studio/pd/application_dashboard/detail_view_response.jsx

```diff
@@ -0,0 +1,39 @@
+import React, {PropTypes} from 'react';
+import _ from 'lodash';
+import {Panel} from 'react-bootstrap';
+
+export default class DetailViewResponse extends React.Component {
+  static propTypes = {
+    question: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
+    answer: PropTypes.any,
+    layout: PropTypes.oneOf(['lineItem', 'panel'])
```

Comment:
Should these be required?
Should we restrict the types of `answer`? It's just boolean, string, and array right?

Note `layout` as it's used now effectively defaults to `panel` so we don't necessarily need to require that one, though then it would be better to use `defaultProps`

Reply:
Question can be required, answer might not be